infra/cifuzz/clusterfuzz_deployment.py

```diff
@@ -85,6 +85,7 @@ class ClusterFuzzLite(BaseClusterFuzzDeployment):
   """Class representing a deployment of ClusterFuzzLite."""
 
   COVERAGE_NAME = 'latest'
+  LATEST_BUILD_WINDOW = 3
```

Comment:
This doesn't seem good to me.
Now we are making assumptions about how often builds are uploaded to support labeling builds by commit.
I think doing this is adding complexity and I'm wondering if the status quo, however flawed is better than this.
Though I guess my current implementation is subject to races (i.e. if a build on a newer commit gets uploaded before build is downloaded).

Reply:
I don't think this is any worse the the current way, which is still prone to races as you mention. This window is just trying to mitigate any potential races from a build for the parent commit not finishing prior to this current job running. 

We can provide a template that includes a job which uploads builds on every push for example.

Reply:
ACK